packages/p2p/src/pages/orders/order-table/order-table-row.jsx

```diff
@@ -203,6 +218,8 @@ const OrderRow = ({ row: order }) => {
                     <Table.Row
                         className={classNames('orders__mobile', {
                             'orders__mobile--attention': !isOrderSeen(id),
+                            'orders__mobile--highlight': should_highlight,
```

Comment:
can't we use `order-table-row__highlight` here?

Reply:
no.. tried it.. it was not working.. half of the styles for this file are in a different files.. the other half is in order-table-row.scss.. i think this would be fixed during the order page refactoring.